content/en/docs/concepts/configuration/manage-resources-containers.md

```diff
@@ -34,16 +34,20 @@ For example, if you set a `memory` request of 256 MiB for a container, and that
 a Pod scheduled to a Node with 8GiB of memory and no other Pods, then the container can try to use
 more RAM.
 
-If you set a `memory` limit of 4GiB for that container, the kubelet (and
-{{< glossary_tooltip text="container runtime" term_id="container-runtime" >}}) enforce the limit.
-The runtime prevents the container from using more than the configured resource limit. For example:
-when a process in the container tries to consume more than the allowed amount of memory,
-the system kernel terminates the process that attempted the allocation, with an out of memory
-(OOM) error.
+Both `CPU` and `memory` limits are applied by the kubelet (and
+{{< glossary_tooltip text="container runtime" term_id="container-runtime" >}}),
+and are ultimately enforced by the kernel. On Linux nodes, limits are enforced
+by {{< glossary_tooltip text="cgroups" term_id="cgroup" >}}). However, `CPU` and `memory` limits
+are enforced differently.
 
-Limits can be implemented either reactively (the system intervenes once it sees a violation)
-or by enforcement (the system prevents the container from ever exceeding the limit). Different
-runtimes can have different ways to implement the same restrictions.
+`CPU` limits are enforced by CPU throttling. When a container approaches
```

Comment:
Can we link a follow-up resource on how CPU throttling works in linux?

Reply:
I removed the follow-up resource. I don't think changing this line is necessary. See this [review](https://github.com/kubernetes/website/pull/46222/files#r1793695009).

Reply:
> Can we link a follow-up resource on how CPU throttling works in linux?

I can't find a good one that isn't on some companies website. the cgroup documentation doesn't really go into depth on how it works. Any suggestions?

Reply:
I previously used this one from [Wikipedia](https://en.wikipedia.org/wiki/Dynamic_frequency_scaling) since its open source.

Reply:
hm maybe I'm using throttling wrong in this case. That article looks like it's describing behavior of the hardware of scaling its density up and down to match workloads. In this case, this is really CPU time slicing, where a process gets slices of the CPU time according to its weight, and is limited in going above that CPU time limit